sources/packages/backend/libs/integrations/src/services/disbursement-schedule/e-cert-calculation/e-cert-pre-validation-service-models.ts

```diff
@@ -5,6 +5,49 @@ import {
 import { ProcessSummary } from "@sims/utilities/logger";
 import { EntityManager } from "typeorm";
 
+const ACCEPT_ASSESSMENT_BLOCKING_VALIDATIONS = [
+  ECertFailedValidation.DisabilityStatusNotConfirmed,
+  ECertFailedValidation.MSFAACanceled,
+  ECertFailedValidation.MSFAANotSigned,
+  ECertFailedValidation.HasStopDisbursementRestriction,
+  ECertFailedValidation.NoEstimatedAwardAmounts,
+];
+
+/**
+ * Result of validations for a e-Cert generation.
+ */
+export class ECertPreValidatorResult {
```

Comment:
Created the object to provide the validations results with more information than only the array of failed validations.
It makes it possible to have the logic for `canAcceptAssessment` centralized and, even being only needed for the API, right now, having it here seems to make more sense.